src/main/steps/review-pay-submit/statement-of-truth/content.ts

```diff
@@ -0,0 +1,154 @@
+  applicant2SotFullName: "Enter the second applicant's full name (if applicable) (in Welsh)",
+  confirm: 'Confirm (in Welsh)',
+  errors: {
+    applicant1IBelieveApplicationIsTrue: {
+      required: 'Confirm your statement of truth (in Welsh)',
+    },
+    applicant1SotFullName: {
+      required: 'Enter a full name (in Welsh)',
+    },
+    applicant2SotFullName: {
+      required: 'Enter a full name (in Welsh)',
+    },
+  },
+});
+
+export const form: FormContent = {
+  fields: userCase => {
+    return {
+      ...(userCase.applyingWith === ApplyingWith.ALONE
+        ? {
+            applicant1IBelieveApplicationIsTrue: {
+              type: 'checkboxes',
+              label: l => l.label,
+              labelSize: 'l',
+              section: l => l.section,
+              values: [
+                {
+                  name: 'applicant1IBelieveApplicationIsTrue',
+                  label: l => l.applicant1IBelieveApplicationIsTrue,
+                  value: Checkbox.Checked,
+                },
+              ],
+              validator: isFieldFilledIn,
+            },
+            applicant1SotFullName: {
+              type: 'text',
+              classes: 'govuk-label govuk-!-width-two-thirds',
+              label: l => l.applicant1SotFullName,
+              labelSize: null,
+              validator: isFieldFilledIn,
+            },
+          }
+        : {
+            applicant1IBelieveApplicationIsTrue: {
+              type: 'checkboxes',
+              label: l => l.label,
+              labelSize: 'l',
+              section: l => l.section,
+              values: [
+                {
+                  name: 'applicant1IBelieveApplicationIsTrue',
+                  label: l => l.applicant1IBelieveApplicationIsTrue2,
+                  value: Checkbox.Checked,
+                },
+              ],
+              validator: isFieldFilledIn,
+            },
+            applicant2IBelieveApplicationIsTrue: {
+              type: 'checkboxes',
+              label: l => l.label,
+              labelSize: 'l',
+              section: l => l.section,
+              values: [
+                {
+                  name: 'applicant2IBelieveApplicationIsTrue',
+                  label: l => l.applicant2IBelieveApplicationIsTrue,
+                  value: Checkbox.Checked,
+                },
+              ],
+            },
+            applicant1SotFullName: {
+              type: 'text',
+              classes: 'govuk-label govuk-!-width-two-thirds',
+              label: l => l.applicant1SotFullName,
+              labelSize: null,
+              validator: isFieldFilledIn,
+            },
+            applicant2SotFullName: {
+              type: 'text',
+              classes: 'govuk-label govuk-!-width-two-thirds',
+              label: l => l.applicant2SotFullName,
+              labelSize: null,
+              validator: isFieldFilledIn,
+            },
+          }),
```

Comment:
This logic can be simplified to avoid duplication
```suggestion
    return {
            applicant1IBelieveApplicationIsTrue: {
              type: 'checkboxes',
              label: l => l.label,
              labelSize: 'l',
              section: l => l.section,
              values: [
                {
                  name: 'applicant1IBelieveApplicationIsTrue',
                  label: l => l.applicant1IBelieveApplicationIsTrue2,
                  value: Checkbox.Checked,
                },
              ],
              validator: isFieldFilledIn,
            },
            ...(userCase.applyingWith === ApplyingWith.ALONE ? [
              applicant2IBelieveApplicationIsTrue: {
                type: 'checkboxes',
                label: l => l.label,
                labelSize: 'l',
                section: l => l.section,
                values: [
                  {
                    name: 'applicant2IBelieveApplicationIsTrue',
                    label: l => l.applicant2IBelieveApplicationIsTrue,
                    value: Checkbox.Checked,
                  },
                ],
              } ]:[ ]),
            applicant1SotFullName: {
              type: 'text',
              classes: 'govuk-label govuk-!-width-two-thirds',
              label: l => l.applicant1SotFullName,
              labelSize: null,
              validator: isFieldFilledIn,
            },
             ...(userCase.applyingWith === ApplyingWith.ALONE ? [
                applicant2SotFullName: {
                  type: 'text',
                  classes: 'govuk-label govuk-!-width-two-thirds',
                  label: l => l.applicant2SotFullName,
                  labelSize: null,
                  validator: isFieldFilledIn,
                },] :[ ])
          }),
```

Reply:
Currently not feasible for me to optimise the logic , will do later

Reply:
Ok no problem